package.json

```diff
@@ -66,11 +66,11 @@
   "devDependencies": {
     "@helpscout/hsds-react": "^2.5.6",
     "@helpscout/zero": "^2.0.5",
```

Comment:
Would it be worth it to update to Zero 3?

Reply:
@tinkertrain Zero@3 broke build of Fancy :(

https://github.com/helpscout/fancy/issues/42

I have a working fix for it :). Tracking the issue down took a while, but the solution isn't too bad